### PR TITLE
Images

### PR DESCRIPTION
**Pull request by krate27**
Preview:

Screenshot (537)
Screenshot (538)

### CODE REVIEW

File: Components/Images.css [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+.ul-image,
+.li-image {
+    list-style: none;
+    margin: 0;
+    padding: 0;
+}
+.ul-image { display: -webkit-flex; display: flex; text-align: center; justify-content: space-between; margin-bottom: 30px; }
+
+.ul-image .li-image { width: 15%; text-align: center; position: relative; padding-bottom: 15%; border-radius: 50%; overflow: hidden; }
+.ul-image .li-image .img-component { position: absolute; width: 100%; height: 100%; top: 0; left: 0; object-fit: cover; }
```
Comment (Hamdrive) [inline Components/Images.css:22]: Check your code formatting structure, make sure it doesnt affect readability

File: Components/input.css [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    display: flex;
+    flex-direction: column;
+    gap: 20px;
+    max-width: 1000px;
+}
+
+.input-box {
+    padding: .5em 0;
+    margin: 10px;
+    width: 30VW;
```
Comment (Hamdrive) [inline Components/input.css:11]: units should be smallcase

**Review by Hamdrive — CHANGES_REQUESTED** [review]
Code look good, will suggest to look into fixing your code formatter
Also comments made

**Review by ZuberDunge — APPROVED** [review]
Apart from what hamza suggested, rest looks fine to me. Good to go from my side.

**Pull request merged**